dashboard/client/src/pages/job/JobRow.tsx

```diff
@@ -56,6 +57,29 @@ export const JobRow = ({
     }
   })();
 
+  const logsLink = (() => {
+    let link: string | undefined = undefined;
+    if (driver_agent_http_address) {
+      link = `/log/${encodeURIComponent(
+        `${driver_agent_http_address}/logs`,
+      )}?fileName=${type === "DRIVER" ? job_id : `driver-${submission_id}`}`;
+    } else if (driver_info && ipLogMap[driver_info.node_ip_address]) {
```

Comment:
Are there instances where both cases can be true (both fields are specified)? Wouldn't the two endpoints be the same or show equivalent info?

Reply:
yes, I assume they will lead to the same place so I just pick one to prioritize.